### PR TITLE
[FEATURE] Implement dark theme

### PR DESCRIPTION
**Pull request by florindakinandcarta**

### CODE REVIEW

File: app/src/main/res/drawable-night/rectangle.xml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shape="rectangle" >
+
```
Comment (liridon-kin-and-carta) [inline app/src/main/res/drawable-night/rectangle.xml:4]: Please delete this empty line.

File: app/src/main/res/drawable-night/rectangle.xml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+    <!-- Control the width and height of the shape -->
+    <size
+        android:width="200dp"
```
Comment (liridon-kin-and-carta) [inline app/src/main/res/drawable-night/rectangle.xml:8]: Please use dimens values and not hard coded ones.

File: app/src/main/res/drawable-night/rectangle.xml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+        android:height="100dp"/>
+
+    <!-- Control the radius of each corners -->
+    <corners
+        android:topLeftRadius="14dp"
+        android:topRightRadius="14dp"
```
Comment (liridon-kin-and-carta) [inline app/src/main/res/drawable-night/rectangle.xml:14]: Same here, please extract values to dimens file

File: app/src/main/res/values/themes.xml [hunk 4]
```diff
@@ -1,20 +1,12 @@
...
+        <item name="android:textColor">@color/black</item>
     </style>
 
     <style name="Theme.Erabook" parent="Base.Theme.Erabook" />
 
-
-<!--    <style name="moreScreenDividerStyle">-->
-<!--        <item name="android:layout_width">0dp</item>-->
-<!--        <item name="android:layout_height">1dp</item>-->
-<!--        <item name="background">#989494</item>-->
```
Comment (liridon-kin-and-carta) [inline app/src/main/res/values/themes.xml:18]: Please delete commented lines.

**Review by liridon-kin-and-carta — COMMENTED** [review]
Good job, left some minor comments 👍

File: app/src/main/res/values/themes.xml (hunk 4 above)
Comment (florindakinandcarta) [reply app/src/main/res/values/themes.xml:18]: this one shows that these are the deleted lines

Comment (liridon-kin-and-carta) [reply app/src/main/res/values/themes.xml:18]: You're right, I can see it now 😆

**Review by liridon-kin-and-carta — APPROVED** [review]
👍

**Pull request merged**